removal_from_ansible.rst

```diff
@@ -165,3 +165,57 @@ Re-adding collection to Ansible
 -------------------------------
 
 There is no simplified process. Once the collection has been removed from Ansible Y.0.0, it needs to go through the full inclusion process to be re-added to the Ansible package. Exceptions are only possible if the Steering Committee votes on them. The Steering Committee can approve or deny a fast re-entry without going through the full review process.
+
+Collections not satisfying the Collection Requirements
+======================================================
+
+A collection can be removed from the package if it violates one or more of the `Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ without making enough effort to mitigate these violations.
+
+This section is not applicable to cases of broken or unmaintained collections. Instead, see the corresponding paragraphs of this document.
+
+Identifying and removing a collection
+-------------------------------------
+
+Conditions for removal
+~~~~~~~~~~~~~~~~~~~~~~
+
+#. A collection violates one or more of the `Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_.
+#. Collection maintainers have not fixed the violations and have not released a fixed version of the collection within the time period established by this document.
+
+Process
+~~~~~~~
+
+#. A community person who found a violation creates a `community topic <https://github.com/ansible-community/community-topics/issues>`_.
+#. If SC members generally agree that the violation is present, a community person creates an issue in the collection's repository. The issue contains:
```

Comment:
Why does the SC need to adjudicate whether something constitutes a violation before a community member can create an issue? While the steering committee should be notified of violations, I think only the actual removal process should require a vote.

Reply:
I don't think this is meant as a vote, more that at least two SC members agree that this really is a violation (and not something which should be valid, but the formation of the rules needs to be improved, or a misunderstanding, or ...). Basically a small sanity check whether more than one person who has some knowledge with the rules thinks that they are violated.

Reply:
yes, no vote was implied, just you know something like when you report say concerns in the meeting and couple of SC say "yep, it seems to be a violation", that's enough to start the process (to save issue reporter's and others' time in case it's a false warning). I'll try to come up with a better wording. thanks!

Reply:
@gotmax23 @felixfontein changed, PTAL

Reply:
Sounds good to me!

Reply:
I just meant I don't think the steering committee needs to be involved with every potential violation. What do you all think about making filling an issue more of a recommendation for when things are unclear, so we don't need extra bureaucracy and issues when it's a cut and dry violation? This requirement wasn't there before, and I'm not convinced it should be added as part of this change.

Reply:
For instance, a few months ago, I found a violation of the licensing guidelines in community.hashi_vault. I just filed https://github.com/ansible-collections/community.hashi_vault/issues/276 and @briantist fixed it. If this policy was in effect then, wouldn't I have had to file a community-topics issue first? Or am I still misunderstanding?

Reply:
Hmm, I was more thinking about cases where not all involved (reporter and collection maintaines) are sure that this is a violation. Maybe it should be formulated so that the SC is only involved if there is disagreement between the reporter and the maintainers that this is a violation, and/or that it needs to be fixed?

Reply:
@gotmax23 @felixfontein good points! Re-formulated, PTAL